src/App.tsx

```diff
@@ -1,60 +1,146 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [checkInput, setCheckInput] = useState(false);
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+  const [visibleTodos, setVisibleTodos] = useState([...todosFromServer]);
+
+  useEffect(() => {
+    if (Boolean(titleInput.trim().length) && Boolean(+userSelect)) {
+      setCheckInput(true);
+    }
+
+    // eslint-disable-next-line no-extra-boolean-cast
+    if (Boolean(titleInput.length)) {
```

Comment:
It will be converted to a boolean by default in the if statement
```suggestion
    if (titleInput.length) {
```